private/buf/bufcli/bufcli.go

```diff
@@ -801,6 +807,29 @@ func NewImageForSource(
 	return bufimage.MergeImages(images...)
 }
 
+// WellKnownTypeImage returns an image for a well known type source file.
+func WellKnownTypeImage(wkpath string) (bufimage.Image, error) {
+	descriptors, err := protoparse.Parser{}.ParseFiles(wkpath)
```

Comment:
`protoparse` is the old compiler. Ideally, we'd figure out a way to make this work through the `bufimagebuild` package, since that consolidates all knowledge of the compiler (and also implements existing support for auto-discovery of the well known imports).